src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -359,7 +359,7 @@ public NodeView()
 
             var runDescription = new Run();
             runDescription.SetBinding(Run.TextProperty, new Binding("Description") { Mode = BindingMode.OneWay });
-            textBlock4.Inlines.Add(runOriginalName);
+            textBlock4.Inlines.Add(runDescription);
```

Comment:
No idea what the difference is between `runOriginalName` and `runDescription`, but it sounds like a tooltip should be some kind of description, so it looks like the right fix.